chat-widget/src/components/postchatsurveypanestateful/PostChatSurveyPaneStateful.tsx

```diff
@@ -64,6 +65,22 @@ export const PostChatSurveyPaneStateful = (props: IPostChatSurveyPaneStatefulPro
         ...props.controlProps
     };
 
+    if (controlProps.surveyURL) {
+        if (!isValidSurveyUrl(controlProps.surveyURL)) {
+            TelemetryHelper.logLoadingEvent(LogLevel.ERROR, {
+                Event: TelemetryEvent.PostChatSurveyUrlValidationFailed,
+                Description: `${controlProps.surveyURL} is not a valid Survey URL`
+            });
+
+            controlProps.surveyURL = "";
+        } else {
```

Comment:
is it really needed?, in the absence of error isn't assumed is correct?

Reply:
We currently do not have telemetry around the survey url used, so logging it would helpful on investigations. Right now, we're only validating the root domain. We could leverage this further to add additional validations in the future as well.